test/serializer/pure-functions/CastStringOnUnknown.js

```diff
@@ -0,0 +1,22 @@
+// additional functions
+// abstract effects
```

Comment:
Hm. Why doesn't this need recover-from-errors? Seems like it would trigger some error?

Reply:
@sebmarkbage I believe we're not trying to detect errors here and the serializer test runner works differently? I didn't need to add it and it definitely went through the same code path as it fails without this PR.